addons/website_event_exhibitor/report/website_event_exhibitor_templates.xml

```diff
@@ -16,4 +16,21 @@
             </div>
         </xpath>
     </template>
+
+    <template id="event_report_template_full_page_ticket_inherit_exhibitor" inherit_id="event.event_report_template_full_page_ticket">
+        <xpath expr="//div[hasclass('o_event_full_page_ticket_sponsors_container')]" position="inside">
+            <t t-set="sponsors" t-value="event.sponsor_ids.filtered(lambda sponsor: sponsor.image_128 and sponsor.website_published)[:3]"/>
+            <div t-if="sponsors" class="o_event_full_page_ticket_sponsors_container row g-0 pt-2">
+                <t t-foreach="sponsors" t-as="sponsor">
+                    <div class="col-4 text-center">
+                        <div class="h-100 p-2 pb-0 o_event_full_page_ticket_sponsor_card d-inline-block">
+                            <!-- Has to be base64-ified otherwise the reporting engine will sometimes load images
+                            and sometimes not before printing, resulting in random results. -->
+                            <img class="img img-fluid" t-attf-src="data:images/png;base64,#{sponsor.image_128}"/>
```

Comment:
are we sure it's always `images/png` ? (not sure what happen if the mimetype there is not correct)

Looks like it work for png / jpeg, but not svg: https://stackoverflow.com/questions/54073217/html-incorrect-media-type-still-correctly-displays-a-base64-image

Maybe it's ok because we do that everywhere in Odoo, just in case you have something cool in mind

Reply:
Yeah png / jpeg seem to work. I retook what existed for sponsors. In a followup task I have pretty much the same issue, especially inside a report, but I did not find a better solution ^^'